Assets/Prefabs/Copilots/HealingBubble.prefab

```diff
@@ -0,0 +1,4879 @@
+%YAML 1.1
```

Comment:
Add a new folder within Copilots for the mushroom friend. Add both that prefab and the healing bubble into it for better organization